schemas/com.snowplowanalytics.mobile/application/jsonschema/1-0-0

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for an application context which automatically tracks version number and build name when using our mobile SDK's.",
+	"self": {
+		"vendor": "com.snowplowanalytics.mobile",
+		"name": "application",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"type": "object",
+	"properties": {
+		"version": {
+			"type": "string",
+			"description": "Version number of the application e.g 1.1.0",
+			"maxLength": 32
+		},
+		"build": {
+			"type": "string",
+			"description": "Build name of the application e.g s9f2k2d or 1.1.0 beta",
+			"maxLength": 32
```

Comment:
I would make these fields longer. All it takes is for somebody to be using SHA1 hashes or something in their version numbers and we will be in trouble. There's no upside to setting an arbitrarily low limit.